src/signals/incident-management/components/FilterForm/__tests__/FilterForm.test.js

```diff
@@ -307,6 +299,25 @@ describe('signals/incident-management/components/FilterForm', () => {
     expect(container.querySelector('input[type="text"][name="address_text"]').value).toEqual('Weesperstraat 113/117');
   });
 
+  it('should watch for changes in note_keyword field value', async () => {
+    const { container } = render(
+      withAppContext(
+        <FilterForm {...formProps} filter={{ name: 'My saved filter', options: { note_keyword: 'test123' } }} />
+      )
+    );
+
+    const noteField = container.querySelector('input[type="text"][name="note_keyword"]');
+
+    act(() => {
+      fireEvent.change(noteField, { target: { value: 'test123' } });
+      fireEvent.blur(noteField);
+    });
+
+    await wait();
```

Comment:
`wait()` Is deprecated in an upcoming version of `@testing-library/react`. Please replace it with a `findBy...` query function, for instance
```jsx
await findByTestId('filterNote');
```

Reply:
:+1: